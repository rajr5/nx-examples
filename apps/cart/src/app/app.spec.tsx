import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render, cleanup } from 'react-testing-library';

import App from './app';

describe('App', () => {
  afterEach(cleanup);

  it('should render successfully', () => {
    const { baseElement } = render(
      <MemoryRouter initialEntries={['/cart']}>
        <App />
      </MemoryRouter>
    );
    expect(baseElement).toBeTruthy();
  });

  it('should have a greeting as the title', () => {
    expect(render(
      <MemoryRouter initialEntries={['/cart']}>
        <App />
      </MemoryRouter>
    ).baseElement).toMatchSnapshot();
  });
});
