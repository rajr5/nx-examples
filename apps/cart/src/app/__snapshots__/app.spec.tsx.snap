// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should have a greeting as the title 1`] = `
<body>
  <div>
    <div>
      Welcome to cart!
    </div>
  </div>
</body>
`;
